blue_hardware_interface/include/blue_hardware_interface/blue_kinematics.h

```diff
@@ -61,6 +62,14 @@ class BlueKinematics
       const std::vector<double> &softstop_max_angles,
       double softstop_tolerance);
 
+  std::vector<double> getPositionActuatorCommands(
+      const std::vector<double> &softstop_min_angles,
+      const std::vector<double> &softstop_max_angles,
+      double softstop_tolerance);
+
+  // Sorry brent
+  int num_joints_;
```

Comment:
oof apology accepted
but maybe we can replace this with `size_t getJointCount() const;` ? and make the member variable private again?